src/js/components/stock-movement-wizard/combined-shipments/AddItemsPage.jsx

```diff
@@ -159,17 +159,49 @@ const FIELDS = {
           },
         }),
       },
-      quantityRequested: {
+      packsRequested: {
         type: TextField,
-        label: 'react.stockMovement.quantity.label',
-        defaultMessage: 'Qty',
+        label: 'react.stockMovement.quantityPOUom.label',
+        defaultMessage: 'Quantity (in PO UoM)',
         flexWidth: '1',
         required: true,
+        headerTooltip: 'react.stockMovement.quantityPerUom.InputTooltip.label',
+        multilineHeader: true,
         attributes: {
           type: 'number',
           showError: true,
         },
       },
+      unitOfMeasure: {
+        type: TextField,
+        label: 'react.stockMovement.POUom.label',
+        defaultMessage: 'PO UoM',
+        flexWidth: 0.9,
+        attributes: {
+          disabled: true,
+        },
+      },
+      calculatedQuantityRequested: {
+        type: TextField,
+        label: 'react.stockMovement.quantityEach.label',
+        defaultMessage: 'Quantity (each)',
+        multilineHeader: true,
+        flexWidth: 1,
+        attributes: {
+          disabled: true,
+        },
+        getDynamicAttr: ({ rowIndex, values }) => ({
+          formatValue: () => {
+            const packsRequested = _.toInteger(
+              _.get(values, `lineItems[${rowIndex}].packsRequested`),
+            );
+            const packSize = _.toInteger(
+              _.get(values, `lineItems[${rowIndex}].packSize`),
+            );
+            return packsRequested * packSize;
```

Comment:
Can you just use `values.lineItems[rowIndex].packsRequested` with nullsafes instead of this lodash function?